### ISSUE TITLE
Multiple github release downloads don't match sha256sum

### ISSUE THREAD

**Issue by pgray** [kubernetes/minikube#1524]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG REPORT

**Minikube version** (use `minikube version`):
`version 0.{10,17,18,19}.0`

**Environment**:
- **OS** (e.g. from /etc/os-release): linux
- **VM Driver** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep DriverName`): n/a
- **ISO version** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep -i ISO` or `minikube ssh cat /etc/VERSION`): n/a
- **Install tools**: `curl`
- **Others**:

**What happened**:
I checked the provided sha files and the sha of the binary and they were different.

**What you expected to happen**:
I expected the sha sums to match

**How to reproduce it** (as minimally and precisely as possible):
Download the linux binary and sha for the above listed releases and check their sha256 sum.

**Anything else do we need to know**:

**Comment by aaron-prindle** [comment]
Thanks for catching this!  It seems that there is only inconsistency with the linux binary.  Definitely need to fix this.  We store another sha256 that minikube checks locally and this value is stored here:
https://github.com/kubernetes/minikube/blob/master/deploy/minikube/releases.json#L6
This value appears to be correct.

If you rebuild the binary you get a different sha256 and I'm guessing that we are referencing two different v0.19.0  binaries in our jenkins release pipeline.  I don't think there is a serious issue here but I definitely understand your concern and I will add a PR to our jenkins pipeline to make sure the values are identical as that is the whole point of it.

**Comment by TripleDogDare** [comment]
I believe this is still a problem with the v0.20.0 release. I haven't set up a build environment to check reproducibility.
```
$ sha256sum minikube 
f7447a37332879b934bf7fcae97327367a5b92d33d12ea24301c212892efe326  minikube
```
does not match
```
==> minikube-linux-amd64.sha256 <==
f7d51f0b94f88650ced3ddb6e079f560e7ef142b8e4d53a5aee511ca1c2e3df4
```

**Comment by onionjake** [comment]
This is still an issue with v0.20.0.  I downloaded from the github link and the google storage link.

```
[~/bin]$ cat ~/Downloads/minikube-linux-amd64.sha256
f7d51f0b94f88650ced3ddb6e079f560e7ef142b8e4d53a5aee511ca1c2e3df4
[~/bin]$ curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.20.0/minikube-linux-amd64 && chmod +x minikube
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 78.7M  100 78.7M    0     0  7025k      0  0:00:11  0:00:11 --:--:-- 7350k
[~/bin]$ sha256sum minikube
f7447a37332879b934bf7fcae97327367a5b92d33d12ea24301c212892efe326  minikube
[~/bin]$ sha256sum ~/Downloads/minikube-linux-amd64
f7447a37332879b934bf7fcae97327367a5b92d33d12ea24301c212892efe326  /home/user/Downloads/minikube-linux-amd64
```

**Comment by tmjd** [comment]
This looks to be fixed with the v0.21.0 release.
